### PR TITLE
Document umami design

### PR DESCRIPTION
**Pull request by dawehner**
Task: #

* [x] Ready for review
* [x] Ready for merge

... description

### Notes

... Notes-optional

### Test Instructions

Test-instructions

### QA

... Screenshots/Screencasts-optional

### CODE REVIEW

File: docs/demos-consumers-implementation-guidelines.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 
 Consumers MUST implement those [wireframes](https://www.drupal.org/node/2818741#comment-12114776)
 
+## designs
```
Comment (e0ipso) [inline docs/demos-consumers-implementation-guidelines.md:8]: Can we change the terminology to `Wireframes`?

Comment (dawehner) [reply docs/demos-consumers-implementation-guidelines.md:8]: ... But they are designs, not wireframes. I'm confued

Comment (dawehner) [reply docs/demos-consumers-implementation-guidelines.md:8]: We already have a wireframes section.

Comment (e0ipso) [reply docs/demos-consumers-implementation-guidelines.md:8]: Oh, my bad. I clicked in the wrong place.

**Review by e0ipso — APPROVED** [review]

**Comment by dawehner** [comment]
Cool, thank you!

**Pull request merged**